kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -12,22 +27,52 @@
 
 <script>
 
+  const isAdminOrSuperuser = require('kolibri.coreVue.vuex.getters').isAdminOrSuperuser;
+  const learnAppConstants = require('../../../../../plugins/learn/assets/src/state/constants');
+  const pageMode = require('../../../../../plugins/learn/assets/src/state/getters').pageMode;
+  const activeCoach = require(
+    '../../../../../plugins/coach_tools/assets/src/state/getters').activeCoach;
+  const activeManage = require(
+    '../../../../../plugins/management/assets/src/state/getters').activeManage;
+
```

Comment:
when we talked about this yesterday, I'd suggested using props to specify which page is currently active.

Either way we have to have some knowledge of the plugins, but this is actually leaking information about the plugins internal state and module structure which is much smellier